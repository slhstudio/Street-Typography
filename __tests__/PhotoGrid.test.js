import React from 'react';
import PhotoGridComponent from '../client/components/PhotoGrid';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

const photos = ['photo1.png', 'photo2.png', 'photo3.png'];
const notes = ['notes1', 'notes2', 'notes3']

describe('<PhotoGrid/>', () => {
  it ('renders and displays properly', () => {
    const wrapper = shallow(<PhotoGridComponent photos={photos} notes={notes}/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});