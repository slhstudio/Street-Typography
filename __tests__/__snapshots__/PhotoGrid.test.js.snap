// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PhotoGrid/> renders and displays properly 1`] = `
<ul
  className="photoGrid"
>
  <li
    key="0"
  >
    <Link
      replace={false}
      to="/photo/photo1.png"
    >
      <img
        alt="street typography image: notes1"
        className="photoItem"
        src="/uploads/photo1.png"
      />
    </Link>
  </li>
  <li
    key="1"
  >
    <Link
      replace={false}
      to="/photo/photo2.png"
    >
      <img
        alt="street typography image: notes2"
        className="photoItem"
        src="/uploads/photo2.png"
      />
    </Link>
  </li>
  <li
    key="2"
  >
    <Link
      replace={false}
      to="/photo/photo3.png"
    >
      <img
        alt="street typography image: notes3"
        className="photoItem"
        src="/uploads/photo3.png"
      />
    </Link>
  </li>
</ul>
`;
